web-app/js/portal/ui/Viewport.js

```diff
@@ -7,6 +7,21 @@
 
 Ext.namespace('Portal.ui');
 
+window.onbeforeunload = function() {
+
+    if (   Portal.config.isFeatureToggleEnabled('chatOnFailedSearch')
```

Comment:
It's to align with the next line... deliberate, but perhaps not in line with the standard.